tsl/src/continuous_aggs/common.c

```diff
@@ -214,6 +221,152 @@ process_additional_timebucket_parameter(ContinuousAggsBucketFunction *bf, Const
 	}
 }
 
+static void
+process_timebucket_parameters(FuncExpr *fe, ContinuousAggsBucketFunction *bf,
+							  AttrNumber htpartcolno, bool process_checks, bool is_cagg_create)
+{
```

Comment:
Would be good to have a comment for the function that explains in what way it processes the parameters. Does it check them? Save them away somewhere? Something else?

Reply:
Added!